src/components/WDLViewer.js

```diff
@@ -1,5 +1,12 @@
+/* global Prism */
+import 'prismjs';
+import 'prismjs/components/prism-bash';
+import 'prismjs/components/prism-python';
+import 'prismjs/plugins/line-numbers/prism-line-numbers';
+import 'prismjs/plugins/line-numbers/prism-line-numbers.css';
+import 'prismjs/themes/prism.css';
```

Comment:
These used to be added by a Babel plugin configured in config-overrides.js. Since the build process doesn't use Babel anymore, they had to be imported here. This is how they were originally imported, before https://github.com/DataBiosphere/terra-ui/pull/1695.